CMakeLists.txt

```diff
@@ -744,6 +745,7 @@ elseif(ANDROID)
         sign
         --ks ${ANDROID_KEYSTORE}
         ${ANDROID_APKSIGNER_KEYSTORE_PASS} ${ANDROID_KEYSTORE_PASS}
+        $<$<BOOL:${ANDROID_KEY_PASS}>:--key-pass> $<$<BOOL:${ANDROID_KEY_PASS}>:${ANDROID_KEY_PASS}>
```

Comment:
It would be nice to use a generator expression for the `--ks-pass` argument above as well

Reply:
I can't decide if I like it or not, but it IS nice with a oneliner, no? :sweat_smile: I'll set it up.